modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/structures/FieldGroups.testcase

```diff
@@ -32,6 +32,63 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-161853. Verify if all field groups are present after realigning."
+	@priority = 4
+	test CheckFieldGroupsAfterResize {
+
+		task ("Given a Web Structure form with a field group") {
+			DataEngine.addField(
+				fieldFieldLabel = "Text",
+				fieldName = "Text");
+
+			DataEngine.addFieldNested(
+				fieldFieldLabel = "Date",
+				fieldName = "Date",
+				targetFieldLabel = "Text");
+		}
+
+		task ("When the fields in the field group is resized and realigned") {
+			FormFields.viewFieldSize(
+				fieldName = "Text",
+				size = 12);
+
+			FormFields.resizeField(
+				fieldName = "Text",
+				moveForce = "-400");
+
+			FormFields.moveFieldToPosition(
+				fieldColumnNumber = 1,
+				fieldLabel = "Date");
+
+			Click(locator1 = "Sidebar#BACK");
```

Comment:
For some reason, the side bar does not navigate back to the fields, so added this to ensure that the third field can be added.